tests/integration/create.bats

```diff
@@ -39,3 +39,19 @@ function teardown() {
 
   testcontainer test_busybox running
 }
+
+@test "runc create --pid-file" {
+  runc create --pid-file pid.txt --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  testcontainer test_busybox created
+
+  # check pid.txt was generated
+  [ -e pid.txt ]
```

Comment:
I'm not sure that it's "not necessary". I don't think we have a test that makes sure that `pid-file` contains the right information? That's why I was suggesting it. But yeah, that change is also fine.